consensus/abci_consensus_connection.go

```diff
@@ -0,0 +1,220 @@
+package consensus
```

Comment:
[MINOR] The name `abci_consensus_connection` is a bit redundant as we are already `consensus`  module and `abci` is already a connection xD. I agree on the separation of the info|state methods, but I would rather keep this method as `abci.go` or `abci_application.go`

Reply:
[IMP] In order to not lose track of changes on this relevant file. Please use `git mv old_filename new_filename`

Reply:
yea didn't know GoLand wasn't smart enough to execute the git mv in the background. Will do next time, good idea.

Reply:
I still feel like it brings value to do the `git mv` on this version. Do you think you could undo and redo your change to rename `abci.go` to `abci_consensus_connection.go` who holds the big majority of the impl. PLS :)